content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -231,6 +235,11 @@ Update a FamilyMemberHistory.
 
     PUT /FamilyMemberHistory/:id
 
+_Implementation Notes_
+
+* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error'. Only existing conditions can be removed; condition id must be provided.
+* An update should only be preformed directly after a FamilyMemberHistory is retrieved. If a condition.id is provided from the get operation, it MUST be provided on the subsequent update and must match the condition.id returned from the get.
```

Comment:
should be mention some how some where that a condition.code must be unique? like u cant add the same condition twice? 

And something about certain relationships only being provided once 

Like they will get unhelpful 400 if they do either of those things

Reply:
Let's add that each item in the list must represent a distinct condition.

I don't think we can provide a full list of which relationships should only be provided once, but I think it's fairly straight forward. So let's leave that one off.

Reply:
<img width="657" alt="Screen Shot 2021-09-22 at 3 36 02 PM" src="https://user-images.githubusercontent.com/56039503/134418028-15390e79-928d-4eaf-9bd7-0fb0086b74e3.png">
I put it here, is that a good place?
https://github.com/cerner/fhir.cerner.com/pull/732/commits/0ec200035c4073e7f7f61541d5d2eb5545e56893#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63R290

Reply:
yea that's fine